web/themes/joinup/prototype/sass/_mixins.sass

```diff
@@ -72,3 +72,7 @@
     -ms-transform: translate(0, -50%)
     transform: translate(0, -50%)
 
+// Margin for form elements
+=form-margin
```

Comment:
Yes, it makes more sense to have a variable and use it in the form component.
My main concern here was that `form-margin` mixin is too specific and not obviously re-usable. A variable is better, please make as you propose.